amethyst_ui/src/action_components.rs

```diff
@@ -38,11 +62,11 @@ impl Component for OnUiActionImage {
 /// Requires `MouseReactive`.
 pub struct OnUiActionSound {
     /// Sound made when this button is hovered over
-    pub(crate) hover_sound: Option<SourceHandle>,
+    pub(crate) _hover_sound: Option<SourceHandle>,
```

Comment:
`#[allow(unused)]`?

Reply:
This will go once I'm done with this, the `OnUi...` structs will be replaced completely by the new `UiAction`s.